sb_vision/tokens.py

```diff
@@ -110,9 +106,6 @@ def update_3D_transforms(
         self.cartesian = translation
 
         # Polar co-ordinates in the 3D world, relative to the camera
-        self.polar = cartesian_to_legacy_polar(self.cartesian)
-        self.legacy_polar = cartesian_to_legacy_polar(self.cartesian)
```

Comment:
We should ensure that the `robotd` changes to remove `legacy_polar` happen _before_ this change is merged.